app/assets/javascripts/custom.js

```diff
@@ -36,13 +53,13 @@ $(document).ready(function(){
     $(".qty").each(function(){
       _totalItemsInCart += parseInt($(this).val());
     });
-    $("#cart").html('<i class="material-icons left">shopping_cart</i>' + _totalItemsInCart);
+    $("#cart").html("<i class="material-icons left">shopping_cart</i>" + _totalItemsInCart);
```

Comment:
Line is too long.<br>Expected ')' and instead saw 'material'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.